plugins/source/azure/resources/services/cosmosdb/sql_databases.go

```diff
@@ -1,144 +1,89 @@
+				Name:     "cq_id_parent",
+				Type:     schema.TypeUUID,
+				Resolver: schema.ParentIdResolver,
 			},
 			{
-				Name:        "database_users",
-				Description: "A system generated property that specifies the addressable path of the users resource.",
-				Type:        schema.TypeString,
-				Resolver:    schema.PathResolver("SQLDatabaseGetProperties.Resource.Users"),
+				Name:     "resource",
+				Type:     schema.TypeJSON,
+				Resolver: schema.PathResolver("Resource"),
 			},
 			{
-				Name:        "sql_database_get_properties_throughput",
-				Description: "Value of the Cosmos DB resource throughput or autoscaleSettings",
-				Type:        schema.TypeInt,
-				Resolver:    schema.PathResolver("SQLDatabaseGetProperties.Options.Throughput"),
+				Name:     "options",
+				Type:     schema.TypeJSON,
+				Resolver: schema.PathResolver("Options"),
 			},
 			{
-				Name:        "autoscale_settings_max_throughput",
-				Description: "Represents maximum throughput, the resource can scale up to.",
-				Type:        schema.TypeInt,
-				Resolver:    schema.PathResolver("SQLDatabaseGetProperties.Options.AutoscaleSettings.MaxThroughput"),
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("ID"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
 			},
 			{
-				Name:        "id",
-				Description: "The unique resource identifier of the ARM resource.",
-				Type:        schema.TypeString,
-				Resolver:    schema.PathResolver("ID"),
+				Name:     "name",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("Name"),
 			},
 			{
-				Name:        "name",
-				Description: "The name of the ARM resource.",
-				Type:        schema.TypeString,
+				Name:     "type",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("Type"),
 			},
 			{
-				Name:        "type",
-				Description: "The type of Azure resource.",
-				Type:        schema.TypeString,
+				Name:     "location",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("Location"),
 			},
 			{
-				Name:        "location",
-				Description: "The location of the resource group to which the resource belongs.",
-				Type:        schema.TypeString,
-			},
-			{
-				Name:        "tags",
-				Description: "Resource tags.",
-				Type:        schema.TypeJSON,
+				Name:     "tags",
+				Type:     schema.TypeJSON,
+				Resolver: schema.PathResolver("Tags"),
 			},
 		},
 	}
 }
 
-// ====================================================================================================================
-//                                               Table Resolver Functions
-// ====================================================================================================================
+func fetchCosmosDBSQLDatabases(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
```

Comment:
This is a relation of `accounts` now so we first get the accounts resource then the databases resource